pom.xml

```diff
@@ -1,32 +1,51 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  Copyright © 2017-2019 Cask Data, Inc.
+
+  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+  use this file except in compliance with the License. You may obtain a copy of
+  the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+  License for the specific language governing permissions and limitations under
+  the License.
+  -->
+
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
   xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
   <modelVersion>4.0.0</modelVersion>
-  <groupId>co.cask</groupId>
+  <groupId>io.cdap.plugin</groupId>
   <artifactId>cobol-to-avro-transform</artifactId>
   <packaging>jar</packaging>
   <version>1.0.0</version>
   <name>Cobol to Avro Transform</name>
 
   <properties>
     <avro.version>1.7.7</avro.version>
-    <cdap.version>4.3.1</cdap.version>
+    <cdap.version>6.0.0-SNAPSHOT</cdap.version>
     <guava.version>18.0</guava.version>
     <janino.version>3.0.7</janino.version>
     <legstar.avro.version>0.4.2</legstar.avro.version>
     <logback.version>1.2.3</logback.version>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <!-- this is here because project.basedir evaluates to null in the script build step -->
-    <main.basedir>${project.basedir}</main.basedir>
+    <widgets.dir>widgets</widgets.dir>
```

Comment:
are lines 35 and 36 needed?
I looked at the following two repos as sample, and they don't declare them:
https://github.com/data-integrations/google-cloud
https://github.com/data-integrations/wrangler

Reply:
yeah they default to the same values. I think I copy pasted this from somewhere else. Will remove